src/ceph.in

```diff
@@ -515,6 +520,44 @@ else:
                 return line
 
 
+def do_command(parsed_args, target, cmdargs, sigdict, inbuf, verbose):
+    ''' Validate a command, and handle the polling flag '''
+
+    valid_dict = validate_command(sigdict, cmdargs, verbose)
+    # Validate input args against list of sigs
+    if valid_dict:
+        if parsed_args.output_format:
+            valid_dict['format'] = parsed_args.output_format
+
+        if verbose:
+            print("Submitting command: ", valid_dict, file=sys.stderr)
+    else:
+        return -errno.EINVAL, '', 'invalid command'
+
+    while True:
+        ret, outbuf, outs = json_command(cluster_handle, target=target, argdict=valid_dict,
+                inbuf=inbuf)
+
+        if 'poll' not in valid_dict or not valid_dict['poll']:
+            # Don't print here if it's not a polling command
+            break
+
+        if ret:
+            ret = abs(ret)
+            print('Error: {0} {1}'.format(ret, errno.errorcode.get(ret, 'Unknown')),
+                    file=sys.stderr)
+            break
+        if outbuf:
+            print(outbuf)
```

Comment:
if `outbuf` is human-readable content, we should `print(outbuf.decode('utf-8'))`, this is important in py3.